docs/readmes/orc8r/dev_gateway_registration.md

```diff
@@ -0,0 +1,85 @@
+---
+id: dev_gateway_registration
+title: Gateway Registration
+hide_title: true
+---
+# Gateway Registration
+
+## Motivation
+
+Previously, to register a gateway, the operator would have to gather the gateway information (both the hardware id and the challenge key) through `show_gateway_info.py` script, then provision the gateway with the correct values of `control_proxy.yml` and `rootCA.pem`.
+Then, the operator would have to register each gateway through the NMS UI and validate the gateway registration by running `checkin_cli.py`.  
+
+In efforts to simplify gateway registration, the new registration process only requires two steps: registering the gateway at Orc8r and then running `register.py` at the gateway.  
+
+## Overview
+
+The overview of gateway registration is as follows:
+![gateway_registration_overview](assets/orc8r/gateway_registration_overview.png)
```

Comment:
Github quite recently supports [mermaid-diagrams](https://mermaid-js.github.io).
I do not know, whether DocuSaurus does. Could you check and maybe use those text sytle definitions if possible?

Reply:
I think docusaurus does not looking through this issue: https://github.com/facebook/docusaurus/issues/1258